onnx/common/ir.h

```diff
@@ -875,6 +927,11 @@ friend struct Value;
   const_graph_node_list nodes() const {
     return const_graph_node_list(output_, kNextDirection);
   }
+
+  std::vector<OpSetID>& opset_versions() {
```

Comment:
Can you return a const reference to make sure people don't modify it and break class invariants?
If people need to be able to change it but only one kind of change (say they only need to be able to add stuff), add a member `addOpSetVersion()` specifically only allowing this one kind of change. That makes it easier to uphold class invariants.

Reply:
So I could do this, but it would require removing the appropriate OpSetVersion at each iteration of my converter (or, I guess, at the end).  By contrast, if I just edit the vector, it's simpler there.

Reply:
Will change name of method

Reply:
change the name?